### ISSUE TITLE
Clear selections upon dir change

### ISSUE THREAD

**Issue by qqmyers** [scholarsportal/dataverse-globus#5]
It looks like the selection dialog doesn't keep track of the full path in the right pane and uses the path from the left pane, so if/when that changes, the selections on the right are invalid. causing uploads to fail. Either tracking the path or clearing the selections upon dir change would help avoid that issue.

**Comment by pdurbin** [comment]
Right. For example, I couldn't transfer `/~/CryptoDB/Current_Release/CbaileyiTAMU-09Q1/xml/`. I got "file not found" and `"path": "/~/nullCryptoDB-59_CbaileyiTAMU-09Q1_NCBILinkout_Nucleotide.xml"`

![Screen Shot 2022-09-16 at 12 09 27 PM](https://user-images.githubusercontent.com/21006/190702120-58b4e84f-a49b-4588-b23e-f2caa64241bf.png)

I also reported the "Up one folder" button not working for me. (I'm on Firefox, if it matters.) The work around is to edit the path manually. @lubitchv said it's the same bug so I'm mentioning it here. Here's a screenshot of that button:

![Screen Shot 2022-09-16 at 12 11 56 PM](https://user-images.githubusercontent.com/21006/190702358-63597199-e043-4647-a30d-7b480d22aa47.png)

**Comment by qqmyers** [comment]
That's not this issue. Those are due to the EUPath endpoint not having an absolute path. This is an issue where if, assuming EUPath had an absolute path.  in your pic you selected fasta now, /~/CryptoDB/Current_Release/CbaileyiTAMU-09Q1/fasta/xml/ wouldn't exist and files couldn't be transferred. The right pane either needs to clear or to remember what the path was when it was selected. 

I don't think we have an issue for the absolute path problem yet.
